packages/assets-controllers/src/NftController.ts

```diff
@@ -1497,21 +1504,26 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
    * Refetches NFT metadata and updates the state
    *
    * @param options - Options for refetching NFT metadata
-   * @param options.nfts - Array of nfts
    * @param options.networkClientId - The networkClientId that can be used to identify the network client to use for this request.
    * @param options.userAddress - The current user address
    */
   async updateNftMetadata({
-    nfts,
-    networkClientId,
     userAddress = this.config.selectedAddress,
+    networkClientId,
   }: {
-    nfts: Nft[];
-    networkClientId?: NetworkClientId;
     userAddress?: string;
+    networkClientId?: NetworkClientId;
   }) {
     const chainId = this.getCorrectChainId({ networkClientId });
-    const nftsWithChecksumAdr = nfts.map((nft) => {
```

Comment:
Corresponding to the comment I left above, keeping `nfts` would prevent a breaking change:

``` suggestion
    // We want to update only nfts missing name/image or description
    const nftsToUpdate = nfts.filter(
      (singleNft) =>
        !singleNft.name && !singleNft.description && !singleNft.image,
    );

    const nftsWithChecksumAdr = nftsToUpdate.map((nft) => {